www/hosa04/SP1/CSS/aeropress_styles.css

```diff
@@ -0,0 +1,58 @@
+*{
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+
+}
+
+.main_image{
+    padding-top: 100px;
+    padding-bottom: 100px;
+    background-image: url(/hosa04/SP1/images/aeropress.jpg);
```

Comment:
je potreba pouzit relativni cestu k souborum, jinak se to nebude zobrazovat spravne u me a na webu.